content/shmem_test_nbe.tex

```diff
@@ -0,0 +1,38 @@
+\apisummary{
+    The request test routine provides a method for querying the status
+    of the request.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_test_nbe(shmem_request_t *req, int *status);
```

Comment:
For consistency with `shmem_test`, I suggest that `shmem_test_nbe` return `status` directly from the function and not through a pointer argument.